src/Directory.Build.props

```diff
@@ -58,13 +58,17 @@
     <Product>Microsoft Bot Builder SDK</Product>
     <Copyright>© Microsoft Corporation. All rights reserved.</Copyright>
   </PropertyGroup>
-
-  <PropertyGroup>
-    <AssemblyOriginatorKeyFile>$(SolutionDir)\build\35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
+  
+  <PropertyGroup Condition="'$(Project)'!='Microsoft.PowerFx.Performance.Tests'">
+    <AssemblyOriginatorKeyFile Condition="EXISTS('..\..\..\src\build\35MSSharedLib1024.snk')">..\..\..\src\build\35MSSharedLib1024.snk</AssemblyOriginatorKeyFile>
```

Comment:
>Condition="EXISTS('..\..\..\src\build\35MSSharedLib1024.snk')" [](http://example.com/codeflow?start=31&length=62)

This is part of the repo, when doesn't it exist?

Reply:
BenchMarkNet creates a small project and builds it using this .props file and will not find the file as it's in a totally different folder